components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-profiles/reporting-profiles.component.ts

```diff
@@ -23,6 +23,7 @@ export class ReportingProfilesComponent implements OnInit, OnDestroy {
   layerOneData: any = {};
   layerTwoData: any = {};
   control: any = {};
+  profileFilterStatus = 'all';
```

Comment:
I'm not sure this is of use to you yet until its added in (or if you decide to add it earlier) - just added strong typing that looks like it would support this here: https://github.com/chef/automate/commit/dbb3374e2427320d0241aea7bdf553e33c4f4da2#diff-44b3f8b3bfe76490fb6f906b379bc1a4

Reply:
Added type :+1::skin-tone-3:

Reply:
@SEAjamieD Had to revert, it complained that the type was added with a literal string.